packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -137,4 +137,26 @@ void main() {
       expect(failure.message, contains('Actual: ?:<zero widgets with $customDescription'));
     });
   });
+
+  group('find.descendant', () {
+    testWidgets('fails with a descriptive message', (WidgetTester tester) async {
+      await tester.pumpWidget(new Row(children: [
+        new Column(children: [new Text('foo')]),
+        new Text('bar')
+      ]));
+
+      TestFailure failure;
+      try {
+        expect(find.descendant(ancestor: find.widgetWithText(Column, 'foo'),
+                descendant: find.text('bar')), findsOneWidget);
```

Comment:
i would wrap this as:
```dart
        expect(find.descendant(
          ancestor: find.widgetWithText(Column, 'foo'),
          descendant: find.text('bar'),
        ), findsOneWidget);
```
...or some such, to make it clearer what the nesting is